tests/naive.py

```diff
@@ -1449,35 +1448,42 @@ def get_all_mpdist_profiles(
     s=None,
     mpdist_percentage=0.05,
     mpdist_k=None,
-    mpdist_vect_func=mpdist_vect,
+    T_subseq_isconstant=None,
```

Comment:
Note that I changed the signature of this naive function so that we can pass `T_subseq_isconstant` from the naive function `mpdist_snippet` to this function, and then use it (in this function) to provide values for `T_A_subseq_isconstant` and `T_B_subseq_isconstant` for `mpdist_vect`.

Also, note that I added the non-normalized version of this function to `naive.py`. (See the function `get_all_aampdist_profiles`).

Reply:
I see